packages/devtools/lib/src/memory/memory.dart

```diff
@@ -68,8 +75,10 @@ class MemoryScreen extends Screen with SetStateMixin {
   MemoryChart memoryChart;
 
   CoreElement tableContainer;
+  CoreElement history; // Memory navigation history.
 
-  final DebuggerState _debuggerState;
+  NavigationPath memoryPath = NavigationPath();
+  bool fromMemoryHover = false;
```

Comment:
maybe document what fromMemoryHover means.

Reply:
Done, its to know if we're navigating through history via the hover card.  If not, the history is cleared.